lint-staged.config.js

```diff
@@ -1,5 +1,9 @@
+/** @param {string[]} filenames */
+const buildBiomeCommand = (filenames) =>
+  `biome check --no-errors-on-unmatched --apply ${filenames.join(" ")}`;
+
 const config = {
-  "*": "prettier --write --ignore-unknown",
+  "*.{js,jsx,ts,tsx,cjs,json}": buildBiomeCommand,
```

Comment:
Why don't we just run this against `*`? It looks like the root `lint` command in `package.json` will process everything?